docs/content/api.md

```diff
@@ -13,23 +13,24 @@ This document contains all information on `renature`'s public facing API.
-| `onFrame`                     | `?(progress: number) => void`               | Optional. Supply a callback function that `renature` will execute on every call to `requestAnimationFrame`.                                                                                                                                                   |
-| `onAnimationComplete`         | `?() => void`                               | Optional. Supply a callback function that `renature` will run when your animation has completed. If the animation is unmounted before completing, `onAnimationComplete` _will not_ be called.                                                                 |
-| `disableHardwareAcceleration` | `boolean?`                                  | Optional. Prevent `renature` from applying optimizations that push compatible animations to the GPU.                                                                                                                                                          |
+| Property                      | Type                                                                                               | Description                                                                                                                                                                                                                                                   |
+| ----------------------------- | -------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `from`                        | `{ [key: keyof React.CSSProperties]: any }`                                                        | The CSS property or properties you're animating **from**. Accepts an arbitrary number of key-value pairs.                                                                                                                                                     |
+| `to`                          | `{ [key: keyof React.CSSProperties]: any }`                                                        | The CSS property or properties you're animating **to**. Accepts an arbitrary number of key-value pairs.                                                                                                                                                       |
+| `reducedMotion`               | `{ from: { [key: keyof React.CSSProperties]: any }, to: { [key: keyof React.CSSProperties]: any }` | Optional. The animation configuration to use if the end user prefers reduced motion.                                                                                                                                                                          |
```

Comment:
Thoughts on adding a callout of the default behavior ~
```
Optional. The animation configuration to use if the end user prefers reduced motion. If no configuration is provided the animation will default to being "disabled" by setting both the "from" and "to" values to be the "to" values, meaning no transition will take place.
```
....but you know, with better words :P

Reply:
ah, I now see you also have the call out in the other docs - which is great, but probably wouldn't hurt to double up here

Reply:
Yep, agreed!